modules/backup-etcd.adoc

```diff
@@ -7,6 +7,8 @@
 
 Follow these steps to back up etcd data by creating a snapshot. This snapshot can be saved and used at a later time if you need to restore etcd.
 
+You should only save a snapshot from a single master host, not one from each master host in the cluster.
```

Comment:
Is _one_ a logic error that the translation team talked about yesterday? 

_You should only save a snapshot from a single master host. You do not need a snapshot from each master host in the cluster._ ??

Reply:
Thanks for the suggestion @mburke5678! I wasn't able to watch that presentation but was looking forward to watching the recording.